src/components/title-blog-index/title-blog-index.jsx

```diff
@@ -12,19 +12,27 @@ const TitleBlogIndex = ({ data, refIndexTitles, disappearIndex }) => {
   const [selectLink, setSelectLink] = useState('');
   const [isTopArrow, setIsTopArrow] = useState(false);
   const [isBottomArrow, setIsBottomArrow] = useState(false);
-  const subContainerRef = useRef(null);
-  const viewContainerRef = useRef(null);
+  const linksContainerRef = useRef(null);
+  const indexContainerRef = useRef(null);
+  const [indexHeight, setIndexHeight] = useState(0);
+  const [linksHeight, setLinksHeight] = useState(0);
 
   useEffect(() => {
     window.addEventListener('scroll', getActiveTitle, { passive: true });
-
-    window.addEventListener('resize', checkOverflow);
+    
+    if (linksContainerRef.current && indexContainerRef.current) {
+      const indexHeightTwo = indexContainerRef.current.getBoundingClientRect();
+      const linksHeightTwo = linksContainerRef.current.getBoundingClientRect();
+      setIndexHeight(indexHeightTwo.height);
+      setLinksHeight(linksHeightTwo.height);
+      setIsTopArrow(linksHeight > indexHeight);
```

Comment:
Does all this work when resizing the screen?

Reply:
It is not when I resize the screen, from the beginning it has to adapt to the container. If someone comes with a smaller screen, they have to see the arrows.